src/components/cluster/detail/cluster_detail_node_pools_table.js

```diff
@@ -222,6 +222,18 @@ const FlexWrapperDiv = styled.div`
   button {
     margin-right: 16px;
   }
+  &.no-nodepools {
+    flex-direction: column;
+    justify-content: space-between;
+    height: 147px;
```

Comment:
This is really specific haha

Reply:
You mean the height? Yeah, this is what I found in wireframes, and BTW I forgot to add an image for this: 

<img width="942" alt="Captura de pantalla 2019-10-16 a las 13 07 20" src="https://user-images.githubusercontent.com/14203438/66915401-3aa12a00-f019-11e9-8739-cf2cc28fceab.png">